docs/troubleshooting-guide.md

```diff
@@ -0,0 +1,87 @@
+# OPCT - Troubleshooting Guide
+
+- [Certification Failures](#review)
+- [Troubleshooting](#review-troubleshooting)
+    - [Review Results Archive](#review-archive)
+    - [Do I Need a Dedicated Test Environment](#review-needed-dedicated)
+    - [Cluster Failures](#review-cluster-failures)
+
+## Certification Tests Failures <a name="review"></a>
+
+Under any type of certification test failure, it is recommended to recreate the cluster under test. The certification tests check cluster metrics and logs which are persisted and this could impact subsequent certification tests.
+
+If you already know the reason for a test failure then resolve the problem and re-run the provider certification tool again so a new certification archive is created.
```

Comment:
If you already know the reason for a test failure then resolve the problem, re-install the cluster, and re-run the provider certification tool again so a new certification archive is created.